libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -119,6 +129,46 @@ class PredictiveCacheController @JvmOverloads constructor(
         mapListeners[map] = onStyleLoadedListener
     }
 
+    /**
+     * Create cache controllers for a map instance.
+     * Call when a new map instance is available.
+     *
+     * @param map an instance of [MapboxMap]
+     * @param styles a list of Mapbox style URIs to cache.
+     * If no styles are passed current map's style will be cached.
```

Comment:
If additional style are provided, will the current map style be cached as well? We need to clarify this.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/5068/commits/a42a365c81916810a07c3257e47e3694ba04c7cf